jetty-util/src/main/java/org/eclipse/jetty/util/Pool.java

```diff
@@ -176,14 +231,15 @@ public Entry reserve(int allotment)
      * @param idx the index of the entry to acquire.
      * @return the specified entry or null if there is none at the specified index or if it is not available.
      */
+    @Deprecated
     public Entry acquireAt(int idx)
```

Comment:
This method should go and any caller should be replaced with a Pool using the RR strategy.

Reply:
Yep - waiting for @sbordet to help rework connection pool.